client/signup/steps/plans/index.jsx

```diff
@@ -138,6 +141,7 @@ module.exports = React.createClass( {
 		return (
 			<StepWrapper
 				flowName={ this.props.flowName }
+				goToNextStep={ isSkipPlansTestEnabled() ? this.onSelectPlan : null }
```

Comment:
Looks like `goToNextStep` is looking for a function. Should we send it `noop` instead of `null`? Further, if we simply don't want to send it, maybe `undefined` would be more appropriate.